pom.xml

```diff
@@ -1,407 +1,418 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.pinterest</groupId>
-    <artifactId>secor</artifactId>
-    <version>0.23-SNAPSHOT</version>
-    <packaging>jar</packaging>
-    <name>secor</name>
-    <description>Kafka to s3/gs/swift logs exporter</description>
-    <url>https://github.com/pinterest/secor</url>
+	<groupId>com.pinterest</groupId>
+	<artifactId>secor</artifactId>
+	<version>0.23-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>secor</name>
+	<description>Kafka to s3/gs/swift logs exporter</description>
+	<url>https://github.com/pinterest/secor</url>
 
-    <licenses>
-        <license>
-            <name>Apache License, Version 2.0</name>
-            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
-            <distribution>repo</distribution>
-            <comments>A business-friendly OSS license</comments>
-        </license>
-    </licenses>
+	<licenses>
```

Comment:
I think your indentation is too much, the previous 4-space indentation seems fine.